src/entries/popup/pages/home/NFTs/NFTCollections.tsx

```diff
@@ -43,7 +43,9 @@ export default function NFTCollections({
   testnetMode,
   userChains,
 }: NFTGalleryProps) {
-  const { displayMode, hidden, sections } = useNftsStore();
+  const displayMode = useNftsStore((state) => state.displayMode);
```

Comment:
Is it actually more performant to destructure these in one `useNftsStore` call? I suppose displayMode might be independent, but hidden/sections probably always update together